docs/operation/cluster_and_service_configuration.rst

```diff
@@ -291,6 +291,40 @@ This value can be overriden:
 
 Then :ref:`apply the configuration<csc-prometheus-apply-cfg>`.
 
+Change Storage Size
+"""""""""""""""""""
+
+Prometheus is deployed with a default storage size of 10Gi.
+However sizing may need to be increased if you add additional
+services to monitor through ServiceMonitor CR and/or want to
+increase retention time.
+The storage size can be overriden:
+
+.. code-block:: yaml
+
+   ---
+   apiVersion: v1
+   kind: ConfigMap
+   metadata:
+     name: metalk8s-prometheus-config
+     namespace: metalk8s-monitoring
+   data:
+     config.yaml: |-
+       apiVersion: addons.metalk8s.scality.com
+       kind: PrometheusConfig
+       spec:
+         config:
+           storage_size: 30Gi
```

Comment:
FWIW, K8s-style schemas use `storageSize`, not `storage_size`. Also, `storageCapacity` or similar is likely a better term.